src/cloud-resources/src/crd/materialize.rs

```diff
@@ -128,6 +128,10 @@ pub mod v1alpha1 {
         pub in_place_rollout: bool,
         // The name of a secret containing metadata_backend_url and persist_backend_url.
         pub backend_secret_name: String,
+        // The name of a secret containing a `password` key with credentials
+        // for the mz_system user.
+        // Setting this will enable password authentication.
+        pub external_login_secret_mz_system: Option<String>,
```

Comment:
is there a reason we're using a separate secret for this?

Reply:
from the one that we use to configure the backend urls and license key

Reply:
I think that was to leave open the possibility of having additional secrets stored for other users. I'm not sure how relevant that is now, though.